lib/address_space/ua_variable.js

```diff
@@ -499,6 +502,26 @@ UAVariable.prototype.readAttribute = function (attributeId, indexRange, dataEnco
 
 };
 
+/**
+ * @method historyRead
+ * @param historyReadDetails {HistoryReadDetails}
+ * @param indexRange {NumericRange || null}
+ * @param dataEncoding {String}
+ * @param continuationPoint {ByteString}
+ * @param callback {Function}
+ * @param callback.err
+ * @param callback.result {HistoryReadResult}
+ */
+UAVariable.prototype.historyRead = function (historyReadDetails, indexRange, dataEncoding, continuationPoint, callback) {
+    if (continuationPoint) {
+        //TODO: use continutaion point manager to support continuation of historical reads
+        callback(null, new HistoryReadResult({statusCode: StatusCodes.BadNotSupported}));
+    } else if (typeof this["_historyRead"] != "function") {
```

Comment:
sure, should use underscore here. It's the only thing that you did not fix - could you do that too?